app/controllers/concerns/org_selectable.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# Provides methods to handle the org_id hash returned to the controller
+# for pages that use the Org selection autocomplete widget
+module OrgSelectable
+
+  extend ActiveSupport::Concern
+
+  # rubocop:disable Metrics/BlockLength
+  included do
+    private
+
+    # Converts the incoming params_into an Org by either locating it
+    # via its id, identifier and/or name, or initializing a new one
+    def org_from_params(params_in:)
+      params_in = params_in.with_indifferent_access
```

Comment:
I didn't realise this was a type of hash, I've been slowly driven mad by the string vs symbol key thing for literally years

Reply:
me too! I'm hoping its the default in Rails 5-6.